views/courses/index.js

```diff
@@ -0,0 +1,83 @@
+const {coursesModel} = require('../../models/courses');
+const {modelUtils} = require('../../utils/models');
+
+const createCourse = async(req, res) => {
+	if (req.method === 'GET') {
```

Comment:
write test case where this condition is `true` and another test case where this conditions is `false`